README.md

```diff
@@ -1 +1,108 @@
-PHP Library to perform 3 way merge on associative arrays.
+# Relaxedws/merge [![Build Status](https://travis-ci.org/relaxedws/merge.svg?branch=master)](https://travis-ci.org/relaxedws/merge)
+
+A Library to perform recursive 3 way merge algorithm
```

Comment:
It is already `A library to perform`. Can you please explain where you are pointing?